src/Config/RuntimeConfigLoader.cs

```diff
@@ -170,7 +170,8 @@ public RuntimeConfigLoader(string? connectionString = null)
         options.Converters.Add(new EntityRestOptionsConverterFactory(replaceEnvVar));
         options.Converters.Add(new EntityActionConverterFactory());
         options.Converters.Add(new DataSourceFilesConverter());
-        options.Converters.Add(new EntityCacheOptionsConverterFactory());
```

Comment:
This removal causes unit test errors. need to add this back.

Reply:
Thanks for spotting this. 

Removed this accidentally. Added back now.